packages/cashier/src/components/crypto-transactions-history/crypto-transactions-renderer.tsx

```diff
@@ -130,6 +133,15 @@ const CryptoTransactionsRenderer = observer(({ row: crypto }: TCryptoTransaction
                         <Text as='p' color='prominent' size='xxs' weight='bold'>
                             {localize('Transaction hash')}
                         </Text>
+                        {is_third_party_transaction && (
+                            <Icon
+                                className='crypto-transactions-history__table-tooltip'
+                                data_testid='dt_crypto_transactions_history_table_tooltip_mobile'
+                                onClick={onTooltipClick}
+                                icon='IcHelpCentre'
+                                custom_color='#999999'
```

Comment:
I think we can use `--button-secondary-default` variable here instead.

```suggestion
                                custom_color=var(--button-secondary-default)
```